### DISCUSSION TITLE
SquashFS

### DISCUSSION THREAD

**Discussion by mobab-th** [sepinf-inc/IPED#2150] [Ideas]
Hello,
would it be possible to support the reading of SquashFS images?

**Comment by lfcnassif** [comment]
Hi @mobab-th. Yes, I commented about the possibility of using native 7zip for that here: https://github.com/sepinf-inc/IPED/issues/1837#issue-1864086521

But, at first, I wasn't sure if it would be worth to turn on by default. Could you detail a bit how it could be useful for you?

**Comment by mobab-th** [comment]
I think that Squashfs is an easy way to make logical backups under Linux. The only thing missing is the possibility to analyse it in IPED.

Squashfs is actively maintained, AFF tools are apparently not and logical EWF containers are still not possible under Linux.

**Reply by lfcnassif** [reply]
Got it, thanks. Seems a reasonable alternative for Linux logical images.
